Assets/Mirror/Runtime/Transport/Discovery/Data/DiscoveryInfo.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Net;
+using Assets.Scripts.NetworkMessages;
+
+namespace Mirror
+{
+    public class DiscoveryInfo
```

Comment:
Get rid of this class.

Instead put an EndPoint field in GameBroadcastPacket  and return a GameBroadcastPacket from ReadDataFromUdpClient